test/test_wofpy_soap.py

```diff
@@ -152,29 +162,38 @@ def test_getvariableinfoobject(self):
             result = self.client.service.GetVariableInfoObject(
                 self.network+':'+var_code)
             self.assertNotEqual(result, None)
-            #TODO: test that the var code matches the given var code
+            # TODO: test that the var code matches the given var code
 
     def test_getsiteinfo(self):
-        for site_code in self.known_series: #TODO: every site should have siteinfo, but the swis db is not complete
+        # TODO: every site should have siteinfo,
+        # but the swis db is not complete.
+        for site_code in self.known_series:
             result = self.client.service.GetSiteInfo(
                 self.network+':'+site_code)
             self.assertNotEqual(result, None)
-            #TODO: test that the site code in the result matches the given site code
+            # TODO: test that the site code in the result
+            # matches the given site code.
 
     def test_getsiteinfoobject(self):
-        for site_code in self.known_series: #TODO: every site should have siteinfo, but the swis db is not complete
+        # TODO: every site should have siteinfo,
+        # but the swis db is not complete.
+        for site_code in self.known_series:
             result = self.client.service.GetSiteInfoObject(
                 self.network+':'+site_code)
             self.assertNotEqual(result, None)
-            #TODO: test that the site code in the result matches the given site code
+            # TODO: test that the site code in the result
+            # matches the given site code
 
     def test_getvalues(self):
         for site_code, var_code_list in self.known_series.items():
             for var_code in var_code_list:
-                result = self.client.service.GetValues(self.network+':'+site_code, self.network+':'+var_code)
+                self.client.service.GetValues(
```

Comment:
It seems that the tests only needs to call `self.client.service.GetValues` because nothing is done with the `result`. the other calls to `assert` something with the `result`.